src/mrb_redis.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
-#include <hiredis/hiredis.h>
+#include "hiredis.h"
```

Comment:
When installing hiredis, hiredis create header files into `"#{hiredis_dir}/include"`. So, correct path is `hiredis/hiredis.h`.